### PR TITLE
optstings integration changes, adding help information and project documentation

### PR DESCRIPTION
**Pull request by ILabiak**
added -h option with useful information for users.
also added readme for the project

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -36,23 +72,30 @@ function checkFormat(deadline) {
 }
 
 const readArgs = (args) => {
-  if (args.s) {
+  console.dir(args)
```
Comment (DanilByaliy) [inline index.js:75]: All is well, you just need to remove the unnecessary console.dir

File: index.js [hunk 2]
```diff
@@ -161,6 +204,8 @@ function editTask(index, title, description, deadline) {
 }
 
 function markAsDone(index) {
+  console.dir(index)
```
Comment (DanilByaliy) [inline index.js:207]: also here

**Review by DanilByaliy — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (ILabiak) [reply index.js]: deleted

File: index.js (hunk 2 above)
Comment (ILabiak) [reply index.js]: deleted

**Review by DanilByaliy — APPROVED** [review]

**Pull request merged**